tools/generate-docs-integration-screenshot

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python3
+"""Create or update a webhook integration screenshot using a test fixture."""
+
+# check for the venv
+from lib import sanity_check
+
+sanity_check.check_venv(__file__)
+
+import os
+import sys
+
+TOOLS_DIR = os.path.abspath(os.path.dirname(__file__))
+ROOT_DIR = os.path.dirname(TOOLS_DIR)
+
+sys.path.insert(0, ROOT_DIR)
+from scripts.lib.setup_path import setup_path
+
+setup_path()
+
+os.environ["DJANGO_SETTINGS_MODULE"] = "zproject.settings"
+import django
+
+django.setup()
+
+import argparse
+import subprocess
+from typing import Any, Dict, Tuple
+
+import cairosvg
+import requests
+import ujson
+
+from zerver.models import UserProfile, Message
+from zerver.lib.actions import do_create_user, notify_created_bot
+from zerver.lib.upload import upload_avatar_image
+from zerver.lib.actions import do_change_avatar_fields
+from zerver.lib.integrations import WebhookIntegration, INTEGRATIONS
+from zerver.lib.webhooks.common import get_fixture_http_headers
+
+def create_integration_bot(integration_name: str) -> UserProfile:
+    owner = UserProfile.objects.get(id=10)
```

Comment:
We should probably use `get_user()` so this doesn't break if we ever have the IDs change.